model_evaluation/jigsaw_evaluation_pipeline.ipynb

```diff
@@ -210,31 +206,111 @@
    "outputs": [],
    "source": [
     "# User inputs\n",
-    "PERFORMANCE_DATASET = 'gs://kaggle-model-experiments/resources/toxicity_q42017_test.tfrecord'\n",
-    "LABEL_NAME_TEST_FILE = 'frac_neg' #Name of the label in the performance dataset"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# Define features\n",
+    "TOXICITY_PERFORMANCE_DATASET = 'gs://kaggle-model-experiments/resources/toxicity_q42017_test.tfrecord'\n",
+    "LABEL_NAME_TOXICITY_TEST_FILE = 'frac_neg' #Name of the label in the performance dataset\n",
     "\n",
     "# DECODING\n",
     "decoding_input_features = {\n",
     "  TEXT_FEATURE_NAME: tf.FixedLenFeature([], dtype=tf.string),\n",
-    "  LABEL_NAME_TEST_FILE: tf.FixedLenFeature([], dtype=tf.float32)\n",
+    "  LABEL_NAME_TOXICITY_TEST_FILE: tf.FixedLenFeature([], dtype=tf.float32)\n",
     "}\n",
     "\n",
-    "def input_fn_performance(max_n_examples=None, random_filter_keep_rate=1.0):\n",
+    "def input_fn_performance_toxicity(max_n_examples=None, random_filter_keep_rate=1.0):\n",
     "    res = utils_tfrecords.decode_tf_records_to_pandas(\n",
     "        decoding_input_features,\n",
-    "        PERFORMANCE_DATASET,\n",
+    "        TOXICITY_PERFORMANCE_DATASET,\n",
     "        max_n_examples,\n",
     "        random_filter_keep_rate)\n",
     "    res[TEXT_FEATURE_NAME] = list(map(tokenizer, res[TEXT_FEATURE_NAME]))\n",
+    "    res = res.rename(columns={\n",
+    "        LABEL_NAME_TOXICITY_TEST_FILE: 'label'})\n",
+    "    res['label'] = list(map(lambda x :bool(round(x)), list(res['label'])))\n",
+    "    final = res.copy(deep=True)\n",
+    "    return final"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Civil comments performance dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "CIVIL_COMMENTS_PATH = 'gs://kaggle-model-experiments/resources/civil_comments_data/train.tfrecord'\n",
```

Comment:
just saying, does it make sense to train on toxicity_q42017 but test on civil_comments_data?
maybe it make more sense to do training and testing on these two dataset separately?

Reply:
Good point. This evaluation notebook has now the input_fn for both of these datasets, so it is up to the user to choose the right one.
Your point is relevant and we actually will train on civil_comments from now on (at least for my experiments).

Reply:
Thanks for catching this @ianchou!